nmtwizard/preprocess/preprocess.py

```diff
@@ -537,6 +539,26 @@ def process_input(
             "[%d] %s source input:  %s", threading.current_thread().ident, proc, source
         )
 
+        if source_context is not None:
+            for sc_idx, sc in enumerate(reversed(source_context)):
+                tu.add_source(
+                    sc,
+                    name="context_" + str(sc_idx),
```

Comment:
Maybe use the same format string as the target for consistency:

```suggestion
                    name=f"context_{sc_idx}",
```